YACS-RCOS/yacs.n
# Front end integration

closes Issue#703

Database Changes/Migrations
Created professor, club/organization, and pathway table


Test Modifications
test on the admin page

Test Procedure
1. Start docker
2. go to yacs_web
3. go to localhost/admin
4. go to EditProfessors
5. make testcases
6. check console.log to make sure changes reflect the input you are doing
